docs/src/pages/docs/events.md

```diff
@@ -0,0 +1,200 @@
+# Events
+
+Events are an incredibly powerful feature in Anchor. Rather than polling or refreshing to
+find out if an account's state has changed, events are like callbacks that save space, compute,
+and enable asynchronous programming.
+
+The main downside to events are that they are base64 encoded, and therefore not human readable.
+Despite this tradeoff, the UI can easily decode them, and the user benefits from this 
+compact mode of logging.
+
+There are a couple of ways to use events, which vary somewhat:
+- to provide a return value
+- as a callback or interrupt for event-driven programming
+- as a compact and efficient means of data logging
+
+*Return values*: Here we use an event to immediately provide a return value, such as details related
+to a transaction. Since calls to certain programs only return a tx hash, this may not provide sufficient detail to your UI. An event here can provide a richer experience, by returning values about how a particular transaction was executed, for example.
+
+*As a callback*: you may want to monitor an event and be notified only once a transaction is confirmed.
+Instead of polling / querying the blockchain and looking for an account's values to update, we can build our client to only act when the program has emitted an event, saving on compute resources and fees.
+
+*Cheap storage*: since events are stored in base64, they can be an economical way to store data
+without the trouble of account creation and provisioning for rent-exemption, for example.
+
+## Example
+
+The simplest way to use events is to tack them onto the tail of your program's functions, allowing them to emit
+structured data when the function completes without error. 
+
+Start by creating a new anchor project:
+```bash
+$ anchor init Events
+```
+
+Next, we'll merely modify the program in `src/lib.rs` to add an event, and the accompanying data to the `initialize`
```

Comment:
```suggestion
Next, we will modify the program in `src/lib.rs` to add an event, and the accompanying data to the `initialize`
```
I'm against using words like simply/merely/just in technical writing, as the people coming to read this may very well be new, and we should foster a safe/comfortable environment for them. It's also just an additional word that doesn't add meaning to the sentence.

I also replace we'll -> we will. This is a nit/personal, but I think if you want to serve an international audience, contractions/slang should be kept to a minimum.